src/idpyoidc/server/session/grant_manager.py

```diff
@@ -0,0 +1,292 @@
+"""
+Implements a database with branches with N nodes (SessionInstance instances)
+that ends in a Grant instances.
+"""
+import logging
+from typing import Callable
+from typing import List
+from typing import Optional
+from typing import Union
+
+from idpyoidc.encrypter import default_crypt_config
+from idpyoidc.encrypter import get_crypt_config
+from idpyoidc.message.oauth2 import TokenExchangeRequest
+from idpyoidc.server.session.info import ClientSessionInfo
+from idpyoidc.server.token import handler
+from .database import Database
+from .grant import ExchangeGrant
+from .grant import Grant
+from .info import NodeInfo
+from ..exception import InvalidBranchID
+from ..token.handler import TokenHandler
+
+logger = logging.getLogger(__name__)
+
+
+class GrantManager(Database):
+    parameter = Database.parameter.copy()
+    init_args = ["handler"]
+
+    def __init__(
+            self,
+            handler: TokenHandler,
+            conf: Optional[dict] = None,
+            remember_token: Optional[Callable] = None,
+            remove_inactive_token: Optional[bool] = False,
+    ):
+        self.conf = conf or {
```

Comment:
Following the comment in https://github.com/IdentityPython/idpy-oidc/pull/38/files#r961742567
```
self.conf = {
            "session_params": {
                "encrypter": conf or default_crypt_config(),
                "node_type": ["client", "grant"],
                "node_info_class": {"client": ClientSessionInfo, "grant": Grant}
            }
        }
```

Reply:
and ignore this suggestion

Reply:
So we have this PR and set of other PR in the queue all based on the current develop branch.
I'd like to merge this one before the others.
Anyone against that ??